platforms/documentation/docs/src/docs/userguide/reference/gradle_daemon.adoc

```diff
@@ -212,40 +215,111 @@ The following file will be generated:
 toolchainVersion=17
 ----
 ====
 
-The same properties file can be produced without configuring the task in the build script. Using just a command-line argument:
+See <<#daemon_jvm_discovery,Daemon JVM Discovery>> for an explanation of the URLs generated in this file and how to modify them.
+
+The same properties file can be generated without configuring the task in the build script by using a command-line argument:
 
 ----
 $ ./gradlew updateDaemonJvm --jvm-version=17
 ----
 
-If you run the task without any arguments, and the properties file does not exist, then the version of the current JVM used by the daemon will be used.
-
-NOTE: Gradle only supports the major JVM version and JVM vendor as a criterion.
-Support for other criteria may be added in a future release.
+If you run the task without any arguments, and the properties file does not exist, then the version of the current JVM used by the daemon will be used and the file will be created.
 
 On the next execution of the build, the Gradle client will use this file to locate a compatible JVM installation and start the daemon with it.
 
+NOTE: Gradle currently supports only the major JVM version (`--jvm-version`) and JVM vendor (`--jvm-vendor`) as criteria. Support for additional criteria may be added in a future release.
+
 ==== Specifying a JVM vendor
 
-Like the JVM version, the JVM vendor can be used as criteria to select a compatible JVM installation for the build. When no JVM vendor is specified, Gradle will consider all vendors compatible.
+The JVM vendor, like the JVM version, can be used as a criterion to select a compatible JVM installation for the build.
+If no vendor is specified, Gradle considers all vendors compatible.
+
+By default, running `updateDaemonJvm` to create the `gradle-daemon-jvm.properties` file will not generate a JVM vendor criterion.
+To specify a vendor, either configure it in the build script or pass it on the command line using `--jvm-vendor=<value>`:
+
+----
+$ ./gradlew updateDaemonJvm --jvm-version=17 --jvm-vendor=adoptium
+----
 
-By default, running `updateDaemonJvm` to create the `gradle-daemon-jvm.properties` file will not generate a JVM vendor criterion. You must either explicitly specify a JVM vendor for the `updateDaemonJvm` task in the build script or pass a JVM vendor on the command-line with `--jvm-vendor=<value>`.
+Gradle recognizes some JVM vendor strings as equivalent, such as "Adoptium" and "Temurin."
+You can view the list of recognized vendors by running `gradle help --task updateDaemonJvm`.
 
-Gradle recognizes a small number of JVM vendor strings as special and equivalent. For example, "Adoptium" and "Temurin" are considered the same vendor. You can see the list of special vendors by running `gradle help --task updateDaemonJvm`.
+If the specified vendor is not one of the recognized equivalents, Gradle will match it exactly.
+For example, "My Custom JVM" would require an exact match of the vendor name.
```

Comment:
I have no idea how a space in a vendor would be handled - needs checking

Reply:
Fixed to MyCustomJVM but still needs to be checked.